db/migrate/20170530180114_add_manufacturer_and_fru_to_guest_devices.rb

```diff
@@ -0,0 +1,8 @@
+class AddManufacturerAndFruToGuestDevices < ActiveRecord::Migration[5.0]
+  def change
+    add_column :guest_devices, :manufacturer, :string
+    add_column :guest_devices, :fru, :string
```

Comment:
@Fryguy I will rename the column to "field_replaceable_unit".  It is a string like the "field_replaceable_unit" column in the physical_servers table that's used to identify a FRU. An example value is, "00Y2773".